server/src/controllers/Brutes.ts

```diff
@@ -878,6 +879,7 @@ const Brutes = {
           ...rankOrEvent,
         },
         orderBy: [
+          { ascensions: 'desc' },
```

Comment:
Brutes should only be ordered by ascension if the rank 0 is selected, to prevent higher DB response time